### DISCUSSION TITLE
Is what has been already made usable?

### DISCUSSION THREAD

**Discussion by JaxonYon** [Visualistic-Studios/Media-Manager#83] [Q&A]
Can we clone the repository and start using it?

**Comment by VenomStyx** [comment]
I thought it was ready but it looks like it needs some work, I'll try to put some effort into getting it prepared in the next week or so.

If you'd like you can clone & see if it works for you, it may just be my system. 

```sh
# Clone
git clone https://github.com/Visualistic-Studios/Media-Manager.git
cd Media-Manager

# Virtual Environment & Install
python3 -m venv venv && source v*/b*/a*
pip install -r requirements.txt

# Run Application
streamlit run app.py 

# - OR - #

# Run Server
python3 app_server.py
```

If that ends up working let me know and I'll post more info 🙂

**Comment by VenomStyx** [comment]
Figured out what was happening, streamlit made some major changes in 1.10. 

1.9 is the last version that currently works. I'll have to modify it in the future to adapt to the new version. 

Check [the development branch](https://github.com/Visualistic-Studios/Media-Manager/tree/development), you should be able to simply clone & run using the instructions above. I'll make a guide & release once I've got a few more changes completed.
